psi4/driver/qcdb/vib.py

```diff
@@ -696,81 +696,119 @@ def grouper(iterable, n, fillvalue=None):
             text += """{:^{width}}{:{colsp}}""".format(val, '', width=width, colsp=colsp)
         text += '\n'
 
-        text += """{:{presp}}{:{prewidth}}""".format(
-            '', 'Reduced mass ' + _br(vibinfo['mu'].units), prewidth=prewidth, presp=presp)
+        text += """{:{presp}}{:{prewidth}}""".format('',
```

Comment:
Is it impossible to convert this to the PEP498 format? or is `f"...."` incompatible with the `"""...."""` syntax?

Reply:
I don't think it's impossible. This is just complicated formatting, and I didn't want to test it again with f-strings. :-)